src/qibo_cloud_backends/aws_client.py

```diff
@@ -0,0 +1,141 @@
+from qibo.backends import NumpyBackend
+from qibo.config import raise_error
+from qibo.result import MeasurementOutcomes, QuantumState
+
+import re
+import importlib.util
+import sys
+
+# Import Qiskit packages for transpiler
+from qiskit import transpile
+from qiskit import QuantumCircuit
+from qiskit.transpiler import CouplingMap
+from qiskit.circuit.random import random_circuit
+from qiskit import qasm2
+
+from braket.aws import AwsDevice, AwsQuantumTask
+from braket.circuits import Gate, observables
+from braket.circuits import Circuit as braket_Circuit
+# from braket.device_schema import DeviceActionType
+from braket.devices import Devices, LocalSimulator
+# from braket.parametric import FreeParameter
+# from braket.ir.openqasm import Program
+
+class AWSClientBackend(NumpyBackend):
```

Comment:
A better name would be `BraketBackend`

Reply:
We'll go with BraketClientBackend to be consistent with the QiboClientBackend and QiskitClientBackend. :)